using-mobile-extensions/adobe-campaign-standard/README.md

```diff
@@ -331,3 +331,96 @@ To remove the corresponding mobile app in Campaign Standard, click **Remove from
 Deleting your mobile property in Experience Platform Launch does not automatically delete your Campaign Standard mobile app.
 {% endhint %}
 
+### Handling deeplinks included in Campaign In-App messages
+
+A deeplink can be added to in-app messages delivered from Adobe Campaign. The link can be a website URL such as https://www.adobe.com or an app link which can be used to direct the user to a specific area of your app.
+
+{% tabs %}
+{% tab title="Android" %}
+
+#### Handling in-app message deeplinks on Android
+
+Website URL's are handled without any further action by the app developer. If an in-app message is clicked through and contains a valid URL, the device's default web browser will redirect to the URL contained in the in-app notification payload's `url` key (for fullscreen and alert messages) or `adb_deeplink` key (for local and push notifications). App links present in the notification payload can be handled by setting up URL schemes in the app. For more information about setting URL schemes for Android, see [Create Deep Links to App Content](https://developer.android.com/training/app-links/deep-linking).
+
+{% endtab %}
+
+{% tab title="iOS" %}
+
+#### Handling alert or fullscreen notfication deeplinks on iOS
+
+The default behavior for opening URL's and app links on iOS is to use the Safari browser. This behavior is expected if the included deeplink is a web URL, but it does not allow the app to handle app links. To provide a better customer experience, the AEP SDK provides a URL handler for use with alert or fullscreen notification deeplinks.
+
+#### Objective-C
+
+```objective-c
+[ACPCore registerURLHandler:^BOOL(NSString * _Nullable url) {
+	NSLog(@"Inside registerURLHandler callback, clickthrough url is: %@", url);
+  if([url containsString:@"campaigndemoapp://"]){
+    // handle the app link (parse any data present in the app link and/or redirect to a desired area within the app)
+    return true;
+  }
+  // false is returned when the URL should be opened in Safari
+  return false;
+}];
+```
+
+#### Swift
+
+```swift
+ACPCore.registerURLHandler({ url in
+	print("Inside registerURLHandler callback, clickthrough url is: \(url ?? "")")
+	if url?.contains("campaigndemoapp://") ?? false {
+        // handle the app link (parse any data present in the app link and/or redirect to a desired area within the app)
+   	return true
+  }
+  // false is returned when the URL should be opened in Safari
+  return false
+})
+```
+
+#### Handling local or push notification deeplinks on iOS
```

Comment:
There is quite a bit of text in this section, so I recommend breaking it up like this:

#### Handling local or push notification deeplinks on iOS

When a local or push notification is clicked through, the `didReceiveNotificationResponse` instance method is called with the notification response being passed in as a parameter. For more information, see the Apple developer docs at [userNotificationCenter:didReceiveNotificationResponse:withCompletionHandler:](https://developer.apple.com/documentation/usernotifications/unusernotificationcenterdelegate/1649501-usernotificationcenter?language=objc). 

The deeplink URL can be retrieved from the response object passed into the handler method. An example for retrieving the deeplink URL and loading web links is provided below. For more information about handling app links and setting URL schemes for iOS, see [Defining a Custom URL Scheme for Your App](https://developer.apple.com/documentation/uikit/inter-process_communication/allowing_apps_and_websites_to_link_to_your_content/defining_a_custom_url_scheme_for_your_app).